pkg/apis/batch/types.go

```diff
@@ -253,6 +253,11 @@ type CronJobSpec struct {
 	// This is a pointer to distinguish between explicit zero and not specified.
 	// +optional
 	FailedJobsHistoryLimit *int32
+
+	// The timezone to be used when scheduling the job.
+	// Valid options include anything in the IANA Time Zone database.
+	// +optional
+	Timezone *string
```

Comment:
A few suggestions:

1. Personally, I'd go with just string, no pointer. In this case both empty or nil means not set. No need to have a distinction between them.
2. `TimeZone` instead of `Timezone` these are two words. 
3. Make sure to tie this to `Schedule` field, iow. move it right after it and add appropriate comment. This is because at some point in time we're planning on supporting [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Times), which has its own time zone information and it this can't interfere with it.